test/test_evaluate/test_docstring.py

```diff
@@ -150,3 +159,169 @@ def foobar(x, y):
         assert 'capitalize' in names
         assert 'numerator' in names
         assert 'append' in names
+
+    @unittest.skipIf(numpydoc_unavailable, 'numpydoc module is unavailable')
+    def test_numpy_returns(self):
+        s = dedent('''
+        def foobar(x, y):
+            """
+            Returns
+            ----------
+            int
+            """
+            return x + y
+
+        def bazbiz():
+            z = foobar(2, 2)
+            z.''')
+        script = jedi.Script(s)
+        names = [c.name for c in script.completions()]
+        assert 'numerator' in names
```

Comment:
Checks if numpy docstring return type inference is working (previously this was not implemented)